README.md

```diff
@@ -1,2 +1,57 @@
-# pharmacy-data-etl
+# Pharmacy Data ETL
 ETL to retrieve pharmacy data from NHS Organisations API
```

Comment:
the yarn.lock file didn't change for me on an install, however deleting it and performing an install did update it. I agree with all the comments above and have changed the line/requires style.